polling/infrequent/workflow_test.go

```diff
@@ -0,0 +1,35 @@
+package infrequent
+
+import (
+	"github.com/stretchr/testify/suite"
+	"github.com/temporalio/samples-go/polling"
+	"go.temporal.io/sdk/testsuite"
+	"testing"
+)
+
+type UnitTestSuite struct {
+	suite.Suite
+	testsuite.WorkflowTestSuite
+}
+
+func TestUnitTestSuite(t *testing.T) {
+	suite.Run(t, new(UnitTestSuite))
+}
+
+func (s *UnitTestSuite) Test_InfrequentPollingWorkflow() {
+	env := s.NewTestWorkflowEnvironment()
+	testService := polling.NewTestService(1) // Horrible workaround to avoid a timeout
```

Comment:
I don't think this is needed. I believe the test environment time skipping should allow running this test quickly even with many activity retries.

Reply:
There might be a default max attempts (10 or so?) to prevent spinning, but this could be set to something like 5.